### PR TITLE
Create 2020-06-24-end-of-cycle-exploratory-research.md

### PR DESCRIPTION
**Pull request by billwessel-dfe**
A research entry to act as a template for future research posts into the Design History - first iteration

Other sections that we could include (but that don't fit for this particular entry):

- Hypotheses
- Summary of findings (where easily summarised into less than 10 bullet points probably)

### CODE REVIEW

File: app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+## The context
+
+We know that there are specific activities providers undertake when a recruitment cycle is closing. This research was to understand provider needs for:
+* ending a recruitment cycle
+* handling "edge case" applications
+* planning a new cycle
+* data & reporting to DfE
```
Comment (paulrobertlloyd) [inline app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md]:
```suggestion
* data and reporting to DfE
```

File: app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+[End of cycle Miro board analysis](https://miro.com/app/board/o9J_kqtdx_E=/)
+
+## New user needs identified
+
+[Provider user needs](https://trello.com/b/5m1pMTme/apply-provider-needs-e2e)
+
+New need categories:
+* End of cycle
+  * [Know upcoming dates](https://trello.com/c/w2C5r8Tp/177-i-need-to-know-the-upcoming-key-dates-from-dfe-in-advance-so-that-i-can-manage-my-time)
```
Comment (paulrobertlloyd) [inline app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md:51]: Linking to individual Trello cards like this makes me wonder if we shouldn’t use the Trello API to use our boards to power the user needs we refer to on this site? 🧐

File: app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+* [End of cycle management](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g85c8fec79f_0_157)
+* [Dealing with edge case applications](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g80a8d09ca1_0_28)
+* [Preparing for the next recruitment cycle](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g80a8d09ca1_0_35)
+* [Reporting and collecting data](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g80a8d09ca1_0_42)
+* [COVID-19 Impacts](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g8a6173f9e3_0_128)
+* [Prototype feedback](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/#slide=id.g8a6173f9e3_0_198)
+
+## Relates to
+
+* [Sorting by RBD date](../sorting-by-rbd-date)
```
Comment (paulrobertlloyd) [inline app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md]: We have a mechanism for adding related links, which can be done using the frontmatter, like so:

```
related:
  items:
  - text: Sorting by RBD date
    href: /manage-teacher-training-applications/sorting-by-rbd-date
  - text: Reasons for rejection
    href: /manage-teacher-training-applications/reasons-for-rejection-iteration-3
```

These links will then appear in the sidebar.

**Comment by KaharHus** [comment]
This is great @billwessel-dfe. I'm wondering whether we might want to capture how the research fits into team workflow. I think it'd be helpful for further setting the scene? Maya's helpfully broken this down on candidate UR card template https://trello.com/c/ybA54EPd. We might not need an explicit subsection for it, but worth bearing in mind when writing the context section?

File: app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md (hunk 2 above)
Comment (defong) [reply app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md:51]: image

File: app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  * [Mark a candidate as having dropped out](https://trello.com/c/OVeYrUWL/160-i-need-to-be-able-to-mark-someone-as-having-dropped-out-of-the-course-before-they-start-and-after-they-start-so-that-my-records)
+  * [Reject a candidate with an offer if they drop out](https://trello.com/c/ajX2HNQ3/165-i-need-to-be-able-to-reject-someone-who-has-accepted-an-offer-if-they-drop-out-so-that-my-records-are-up-to-date)
+  * [Reject/withdraw a deferred offer if the candidate drops out](https://trello.com/c/pgdRucyL/176-i-need-to-reject-withdraw-a-deferred-application-so-that-if-the-candidate-drops-out-my-records-are-up-to-date)
+* Manage data
+  * [Summary statistical data](https://trello.com/c/ErAkkfqg/180-i-need-data-about-all-my-applications-received-rejected-and-offered-throughout-the-year-so-that-i-can-report-accurate-data)
+  * [Data to support DTTP or HESA](https://trello.com/c/7CxOJ0YW/181-i-need-to-receive-the-correct-data-in-order-to-complete-my-hesa-and-dttp-returns-to-the-dfe)
+
+## Findings report
+
+[End of cycle resarch findings](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/edit?usp=sharing)
```
Comment (defong) [inline app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md]:
```suggestion
[End of cycle research findings](https://docs.google.com/presentation/d/1Jtchf-8CZv_E75UuNFMl-YR9tD-_RcYOEa3wIpOJzqI/edit?usp=sharing)
```

File: app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md (hunk 2 above)
Comment (billwessel-dfe) [reply app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md:51]: @paulrobertlloyd would that pull through the needs from the board even if it's not fully publicly viewable as @defong has illustrated above?

**Comment by billwessel-dfe** [comment]
> This is great @billwessel-dfe. I'm wondering whether we might want to capture how the research fits into team workflow. I think it'd be helpful for further setting the scene? Maya's helpfully broken this down on candidate UR card template https://trello.com/c/ybA54EPd. We might not need an explicit subsection for it, but worth bearing in mind when writing the context section?

@KaharHus It might even be helpful to have a separate entry on how research fits into the team workflow that we could reference in future research entries

**Comment by KaharHus** [comment]
Most definitely! I don't see why not.

File: app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md (hunk 2 above)
Comment (paulrobertlloyd) [reply app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md:51]: Should be possible, yes. I’ll need to have a closer look at the API. Of course, this begs the question should the boards be publicly viewable anyway (assuming they contain non-identifiable information about participants)?

Comment (billwessel-dfe) [reply app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md:51]: @paulrobertlloyd yeah, there's no reason they shouldn't be publicly viewable by anyone who's reached them via the design history. I can't edit that though as Helen owns the board and that's a constraint of how the shared team boards were set up as best I can tell

**Pull request merged**